### PR TITLE
Fix typo in extension README.md

### PR DESCRIPTION
**Pull request by jogo-**

### CODE REVIEW

**Review by nagilson — APPROVED** [review]

**Comment by nagilson** [comment]
Thanks @jogo-  for spotting that typo and fixing it!

**Review by nagilson — APPROVED** [review]

**Review by SrujanNj — APPROVED** [review]

File: vscode-dotnet-runtime-extension/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Prior to the release of this extension, extension authors had no way of knowing
 
 1. **Duplication of .NET runtimes and slow updates**: Each extension was acquiring its own copy of .NET at ~30mb each.
 2. **Clean up**: When extensions installed .NET in a non-VSCode folder location it was likely to be left behind.
-3. **Servicing and floating versions**: It was difficult to ensure that extensions would use the latest releases, particuarly without re-shipping.
+3. **Servicing and floating versions**: It was difficult to ensure that extensions would use the latest releases, particularly without re-shipping.
```
Comment (SrujanNj) [inline vscode-dotnet-runtime-extension/README.md:13]: Can somebody help me in this issue plssss...//?

Comment (SrujanNj) [reply vscode-dotnet-runtime-extension/README.md:13]: 13 ++

**Review by SrujanNj — CHANGES_REQUESTED** [review]
Check this and please suggest the changes

**Comment by baronfel** [comment]
Thanks again for this.

**Pull request merged**